<?xml version="1.0"?>

<launch>

  <!-- Piksi. -->
  <node pkg="piksi_rtk_gps" type="piksi.py" name="piksi" output="screen">
    <!-- Load default settings -->
    <rosparam file="$(find piksi_rtk_gps)/cfg/piksi_driver_settings.yaml"/>
  </node>

  <!-- Geodetic Survey. -->
  <node pkg="piksi_rtk_gps" type="geodetic_survey.py" name="geodetic_survey" output="screen">
    <remap from="navsatfix" to="piksi/navsatfix_spp"/>
  </node>

</launch>
